src/themes/shared/sizes.ts

```diff
@@ -10,6 +10,7 @@ export const sizes = {
   '4xl': '10rem',
   '5xl': '15rem',
   '6xl': '20rem',
+  '7xl': '30rem',
```

Comment:
We need this new tokes. The designers already updated the figma and the figma-tokens file